---
datecreated: 2023/10/18
aliases: 
course:
---
## Thermodynamics

- Not following the order of the book from now on
- [[Bioenergetics]] 
- [[Central Dogma of Molecular Biology]]
- [[Kinetic Energy]] and [[Potential Energy]]
- Potential is stored energy with the potential to do work
	- Based on position
- Kinetic is energy in action
- ATP as a form a energy 
	- Humans go through billions of ATP per cell

## Types of Energy

- [[Hydroenergy]], using water to generate energy
- [[Wind Energy]], use of wind to power turbines
- [[Thermal Energy]], use of Heat
- [[Electrical Energy]]
- [[Solar Energy]]
- [[Chemical Energy]], chemical reactions that release energy, [[Exothermic Reactions]]
- [[Electromagnetic Radiation]]
	- light energy
	- P32 and P33 used as energy 
	- Ultraviolet energy is used for imaging
	- Visible light is a small range in electromagnetic spectrum
	- Color representation of images produced from other waves are just subjective representations 

## Catabolism and Anabolism

[[Metabolism]] 
- Energy used for [[Catabolism]] and [[Anabolism]]
- Catabolism breaks down larger molecules
	- Energy yielding Metabolism
	- Making energy
	- Breaking bonds is converting to energy
	- ATP1 P2 ~ P3
		- wiggle is high energy bond
	- Breaking off P3 releases energy
	- ADPP + Pi + Energy
	- $\ce{ ATP_{1}P_{2}P_{3} -> ADP_{1}P_{2} + P_{i} + Energy }$
- Anabolism is the synthesis of larger molecules

## Thermodynamics

- Laws of [[Thermodynamics]]
	- First Law
		- Energy can be transformed and converted, but not created or destroyed
		- All energy occurred from big bang
		- Wood having a lot of potential energy
			- Setting it on fire transforms energy
			- Creates [[Thermal Energy]] and [[Electromagnetic Radiation]]
			- Wood just changes form
	- [[Third Law of Thermodynamics]] 
		- [[Kinetic Energy]] of [[Atoms]] and [[Molecules]] is represented as temperature or heat
		- [[Absolute Zero]] is impossible elimination of movement of atoms and molecules
	- Second Law 
		- [[Entropy]] always increases over time
			- PhysicsÂ aÂ thermodynamicÂ quantity representing the unavailability of aÂ system's thermal energy for conversion into mechanical work, oftenÂ interpretedÂ as the degree of disorder or randomness in the system:Â theÂ second law of thermodynamics says that entropy alwaysÂ increasesÂ with timeÂ |Â [count noun]Â :Â Â the sum of the entropies of all theÂ bodiesÂ taking part in the process.Â (Symbol:Â S)
		- Can never be negative in the universe
		- Can be in chemical reactions because of chemical equilibration
		- Entropy is direction
		- Heat moves from hot to cold areas 
		- Entropy is represented as $\ce{ S }$
			- "We don't really know what it is"
			- How to quantify it
			- Only measure differences between forms represented as $\Delta S$
		- Can be seen as the level of disorder of mater has
		- How fast entropy happens is kinetics
- [[Arrow of Time]] 
	- Cannot go backwards in time
	- But can see back into it, looking at stars
	- One of the variable of aging

## Example

- Stack of hay has energy
- Horse eats it
- Horse poops
- Process can't go back wards

## Redox Reactions

- Processes of cells can only move forward
- [[Oxidation Reduction Reactions|Redox Reactions]]
	- [[Oxidation Reduction Reactions]]
	- Losing electrons are called Oxidations
	- Gaining electrons are Reductions
	- Loss of Hydrogen
	- [[Disulfide Bridges]] can be broken with reduction agents 
- Example of redox reaction
	- $\ce{ Fe_{2}O_{3} + 3CO -> 2Fe + 3CO_{2} }$
- Oxidation and Reaction are complimentary
- Food and Oxygen to energy plus water and CO2
	- Food Glucose is transformed into ATP and by product water
		- Waste is CO2 which is released to CO2
		- CO2 to blood
		- Blood to lungs
		- Lungs processes it 
- [[Oxidation Reduction Reactions|Oxidation]]
	- Loss of electrons
	- gain of Oxygen
	- Loss of Hydrogen
- [[Oxidation Reduction Reactions|Reduction]]
	- Gain of electrons
	- Loss of Oxygen
	- The Gain of Hydrogen
- Processes formed during anabolic and catabolic reaction
	- H  [[Enthalpy]] is Heat content
	- S - [[Entropy]]
	- T - [[Temperature]]
	- G - [[Gibbs Free Energy Change]]
	- Can't measure entropy well, but can measure the change of [[Gibbs Free Energy Change]]

$$\ce{ \Delta G = \Delta H -T\Delta S }$$

## Gibs Free Energy

Gibbs free energy is a measure of chemical energy.

All chemical systems tend naturally towards states of minimum Gibbs free energy

- $\Delta H$ (Change in Heat)
	- Exerting heat from a [[Systems]] is [[Exothermic Reactions]]
	- A [[Systems]] Absorbing Heat is [[Endothermic Reactions]]
- Reactions can be [[Exergonic Reactions]] which are spontaneous
	- Can happen on its own
	- Not fast or slow
	- No needed outside intervention 
- [[Endergonic Reactions]] is non spontaneous reactions
	- Not enough heat to cause reaction and addition of heat is required 

Products and reactants are in equilibrium when their Gibbs free energies are equal

A chemical reaction will proceed in the direction of lower Gibbs free energy $\ce{ \Delta Gr<0 }$
So the reaction wont proceed if the reaction produces an increase in Gibbs free energy

$\ce{ \Delta Gr=0 }$
$\ce{ \Delta Gr<0 }$
$\ce{ \Delta Gr>0 }$

$\ce{ \Delta G{f}>0 }$ = Free energy of formation 


- [[Enthalpy]] 
	- Spontaneity is favored when the process is exothermic deltaH<0
- Delta H and Delta S are the two driving forces of reactions
- DElta H < 0 and Delta S > 0 will have a spontaneous reaction
- $\ce{ \Delta H<0 }$

## Making Diamonds

- Graphite -> Diamonds
- 1700 K at 12 gigaPa
- DeltaG for both are similar

[[Gibbs Free Energy Change]] Chart

- Delta H < 0 and DElta S > 0

## ATP and Cellular Work 

[[Chemical Work]] - the pushing of endergonic reactions, which wouldn't ordrinally happen
[[Transport Work]]: Transporting molecules against their chemical gradient
[[Mechanical Work]]: Cillian and flagella moving things

ATP Release free energy, when phosphate bonds are hydrolyzed

DeltaG = -7.3kcal/mol 
$$\Delta G = \frac{-7.3kcal}{mol}$$

Reactions can be sped up with [[Catalysts]] such as [[Enzymatic Proteins]], and their names end with -ase.
Speeds up rates of reactions without being consumed. 

$$\ce{ [S] <=> [P] }$$
Reversible reaction until equilibrium happens, which equals death. 

$Concentration = [C]$
$Concentration of Substrate = [C]$

## Enzyme Catalyst

[[Free Enzymes]] + [[Free Substrates]] = E + S
[[Enzyme Substrate Complex]] = ES
[[Enzyme Product Complexes]] = EP
Free Enzyme + [[Free Products]] = E +P

# Lock and Key Fit

- Enzymes have pockets called [[Active Sites]] and has a bunch of [[Amino Acids]]
	- Active sites Typically polar, basic, and acid acids
		- They are geometric and allows substrates to bind well
		- Two thoughts of cause
			- [[Lock and Key Fit]]
				- Substrate has confirmation specific for active site
				- All amino acids will interact with the related amino acids of the active sites
				- Highly Specific
			- [[Induced Fit Theory]]
				- Not looking for exact fits, but similar fits
				- Some of the substrates can be [[ATP]]
				- [[ATPases]] are enzymes for ATP
			- Antibodies bonding to Antigens is specific
	- Holds on to substrates due to amino acid interactions
- And they fit together like a puzzle with specific molecules
- Structure equals functions 

## DNA Polymerase

Polymerase makes [[Deoxyribonucleic Acid|DNA]]. Form of it shows a cleft where processes can happen.
- [x] Homework will be posted âœ… 2023-11-27

# 10/23/2023 Lecture

- [[Deoxyribonucleic Acid Polymerase]] Polymerizes DNA primers binds with templates and build with dntp
- [[Eznymology]]  
- Structure and function is important with enyzmes
- DNA Primer Template build around the Palm of the DNA polymerase
- Primer Templates and DNTP
- Sometimes there is one template

$\ce{ A<=>B<=>C<=>D<=>E }$

- A = Precursor
	- The first Reagent
		- E = End Product
- Precursor  binds substrate
- Substrate will enter active site
- Product B is temporary and becomes substrate for following step C
- No genes for amino acids
- Need several enzymes to make amino acids
- Other biochemical pathways can get to different steps of the process
- Taking out C will leave increase of C product
- How does [[Enzymes]] work
	- It speeds up reaction
	- It lowers the energy needed for reaction to go forward
	- $\ce{ E_{a} }$ = [[Free Energy of Activation]]
	- It is a endergonic reactions
	- Positive Delta G
- Reactants lower in concentration
- Enzymatic processes can be backwards or fowards until $\ce{ K_{eq} }$ Forward = Reverse 
- Enzymes lower energy of activation
	- Enzyme will bind with high affinity 
	- It changes it confirmation
	- Forces parts to interact

## dNTP Polymerization

$\ce{ E + DNA <=>[K_{1}][K_{-1}] E*DNA <=>[K_{2}][K_{-2}] E*DNA*dNTP <=>[K_{3}][K_{-3}] E^* * DNA*dNTP <=>[K4][K-4] E^* * DNA_{+1} * PP_{i} <=>[K5][K-5] E*DNA_{+1}*PP_{i} <=>[K_{6}][K_{-6}] E+DNA_{+1}}$
- Asterisks is the transition state, from substrate to product

## Catalysis

Direction of [[Catalysis]], how to quantify it? Through [[Absorbance]]. Increase of Absorbance is evidence of product increase.

## Effect of pH on Catalysis

[[Chymotrypsin]] works well at pH of 8
[[Pepsin]] works optimally at a level of 2
[[Cholinesterase]] works well at 6.5+

[[Buffers]] are needed to keep function of [[Enzymes]]

## The Affect of Temperature on Enzymatic Activity

Increase of [[Heat Shock Proteins]]
Increasing heat opens up membranes of cell.
[[Cofactors]] such as 2 Mg^2+
YXDD = Motif C
X = can be any amino acid
[[Invarient]] is DD, 2 Aspartic Acid
It can bond with 

## Coupling

$\ce{ C + D -> A + B }$

ATP needs to be broken to move reaction forward
Heat is used as energy
[[Energy Coupling]] occurs
ATP is used a lot

## Gluconeogenesis 

[[Gluconeogenesis]] is the creation of glucose. 
10 - 15 Bio chemical reactions
Substrates
Products
Energetics

diagram of all cell chemical reactions

## Michaelis Menten Reactions

[[Michaelis Menten Kinetics]] occurs with substrate to product. $\ce{ [S] <=> [P] }$
There is a curve with these reactions. $V_{max}$ 
[[Saturation Points]] all active points are filled
Can only increase with addition of additional active sites

$\ce{ \frac{V_{max}}{2} }$ 

$\ce{ Km }$ is [[Michaelis Menten Constant]]
What does the relationship measure?
Km measures how well enzyme can bind substrate with high affinity and change in to product. It can change based on certain parameters.

Changing the [[Michaelis Menten Constant]], adding inhibitors change constant. [[Competitive Inhibitors]] creates competition for [[Active Sites]]. 


Interferes with active site of enzyme so substrate cannot bind.
[[Noncompetitive Inhibitors]] changes shape of enzyme so it cannot bind to substrate

==[[Allosteric Regulation]] changes geometry of [[Active Sites]] to prevent binding

Relative to regular enzyme function
[[Competitive Inhibitors]] have same $\ce{ V_{max} }$ and different $\ce{ K_{m} }$
[[Noncompetitive Inhibitors]] have same $\ce{ K_{m}}$ and different $\ce{ V_{max} }$

## Cooperativity 

==[[Cooperativity]] 

Alpha 1 O_2 binds to it 
Alpha 2 now has higher affinity for O_2
Beta 2 Affininty increase 
Beta 1 Affinity increase


Hemoglobin transports CO2 and is used with oxygen respiration 

## Negative Feedback Inhibition

[[Negative Feedback Inhibition]] can block enzymes to prevent further creation of products.

## DNA Replication

- [[Deoxyribonucleic Acid|DNA]] Replication
- [[Central Dogma of Molecular Biology]]
	- DNA Replication
	- RNA Transcription
	- Protein Translation

## Cell Types

White blood cells
Sperm
Neurones
Fiberglass

Have same DNA from the same organism 

Instructions from DNA that have [[Genes]], and they are called [[Alleles]]
- Slight difference of DNA comes from parent DNA
Difference is in RNA Transcriptions

[[Deoxyribonucleic Acid|DNA]] uses AGTC
[[Nucleobases]] are [[Adenine]], [[Guanine]], [[Thymine]], and [[Cytosine]]
[[Phosphate Groups]]
[[Deoxyribose]]

1928 Fred Griffith

- [[Griffith's Experiment]]
- S Strain - Smooth
	- Virulent
- R Strain - Rough
	- Avirulent
- Transformation 
- [[Transforming Principle]]

# 10/25/2023 Lecture

## Griffith's Experiment

[[Griffith's Experiment]] 

Boiled S Strain and proved the bacteria died
Heat killed s strained mixed with R strain
Something from S strain entered and combined with R Strain
Transforming principal is still unknown

Changing a [[Phenotypes]] changes inheritance of organism 
Change of phenotype is a change of [[Genotypes]] 

## Avery, McCarty and McCleod Experiment

[[Avery, McCarty and McCleod Experiment]] was similar to [[Griffith's Experiment]]
Heat-treated S strain destroys it
Heat treated S Strained with RNases and Proteases
Heat treated S + R
DNases, heat treated S + R
	Mouse lived during this
	DNase was the determined factor
Proteases, heat treated S + R
	Cannot be S strain because it was destroyed during heat treatment

## Hershey-Chase Experiment

[[Hershey-Chase Experiment]] uses [[Bacteriophages]] called $\ce{ T_{i} }$ and $\ce{ T_{2} }$, viruses that infect [[Bacteria]], it injects its dna in to cell, Lytic means to [[Lysis|Lyse]]

Viruses break apart cells. Double stranded dsDNA in [[Bacteriophages|Phages]]. except at the end. They are single straights. Causites they are cohesive sites.
It forms [[Replicative Forms]]

Viruses [[Lysis|Lyse]] cells apart.

Bacterio phage has Protein, DNA, 

Virus can be dsDNA ssDNA dsRNA ssRNA +/=
Hybrid DNA/RNA

[[Deoxyribonucleic Acid|DNA]] can be labelled with 32P can be used to label the backbone. S-35 Met is used. 
S-35 binds to proteins.

Get blended and phages will be released called [[Ghost Phages]] and spin down. Centrifuged to be reduced into a pellet. It's too light so it needs an ultra centrifuge.  

Radioactive material is in the cell.

## DNA

[[Nucleobases]]

[[Purines]] vs [[Pyrimidines]]

Alpha Phosphate gets cleaved

Memorize linkages from bases  to riboses
Take a base like Andenine, 
Cylic AMP - Signaling
[[cyclic AMP]]

## Edwin Chargaff

[[Erwin Chargaff]] 1905 - 2002
Isolated DNA from many organisms 
- Ox
- Human
- Hen
- Salmon
- Wheat
- Yeast
- Hemophilus
- E-Coli K2
Avian Tubercle
Bacillus
Serratia Marcescens
Bacillus

Compared concentration of base pairs
ADENINE TO THYMINE
GUANINE TO Cytosine 
are similar 

## Kings College

[[John Randall]]
- [[Maurice Wilkins]]
	- Biochemist
	- Post Doc
	- Purified DNA
	- Really pure
- [[Rosalind Franklin]]
	- Crystolographic Prodigy
	- Meticulous
	- Post Doc
	- Wanted to work on DNA Structure
[[Raymond Gosling]]

Phd teaches to be a scientist
post doc teaches to be individual

Franklin used Wilkin's pure DNA
Gosling's was not that pure

## Watson and Crick

50 - 60 mi from Kings

[[Linus Pauling]] was an organic chemist
Thought the back bone was along an axis
[[Crystallography]]
[[Image 51]]

Watson from uni of Chicago
Double Helix from image 51
What form are the phosphates

AT Base pair
GC Pair
1:1 to purine to pyradine 

[[Watson Crick Base Pairing]]
Natural form of double helix makes flexible molecules
Non Polar interactions creates helixes
Creates Major and Minor Grooves

## Nucleotide Base Pairing

Denature and come apart and act as templates
Original [[Parent Strands]] bind with [[Daughter Strands]]
A pitch is a full turn of a [[Double Helix]]
~10 bases per pitch
Most DNA molecules are in Right handed helical form
Franklin was able to crystalize two forms
Middle form is [[B-Deoxyribonucleic Acids]]
	Physiological DNA has all parameters
[[A-Deoxyribonucleic Acids]] form pulls out water and twists tighter
	Dehydrated form of DNA
Gosling DNA had more water 

[[Reverse Transcriptase]]

[[Alexander Rich]]

Took a few sequences
$\ce{ 5' - C_{p}G_{p}C_{p}G_{p}G_{p} }$
[[CPG Islands]] form difference helices
[[Z-Deoxyribonucleic Acids]]
- Completely Different DNA
- Bases move away axis
- Zig Zag Conformation
- Left Handed DNA
- Occurs for short amount of time
- Regulatory Role
[[A-Deoxyribonucleic Acids]] with RNA RNA interaction and RNA/DNA Interation
[[Hybrid DNA]]

## Various Conformations of DNA

[[Deoxyribonucleic Acid|DNA]] 

- Understasnd helical sense
- bp oper turn
- angstron helix rise per base pair

DNA can be bent and not break

## DNA Structural Parametrers 

Replication Fork

# 10/30/2023 Lecture

## Making of Helix

- [x] Memorize proteins âœ… 2023-11-27

[[DNA Replication]] 

3'-5'
5'-3'

[[Origins of Replication]], 
ORI C, Prokaryotic 
ORI, Eukaryotic 


[[Plasmids]] are autonomous and replicate on their own, not directed by Genome 
Shuttle vector can go between pro and euk
Prokar.
Eukar.

Ori C. 

Something in euk needs an origin to replicate

Viruses replicate in cells and use cell's substances
[[Viral Origin of Replication]]

Position on genome that initates dna replication
3'-5'
5'-3'

Denature a specific part of DNA strand.

[[Helicase]], type of [[Denaturation Agents]], breaks [[Hydrogen Bonds]] 
Forms [[Replication Bubbles]], which causes formation of [[Replication Forks]] on the 5' side.
Detects specific sequences
Prokaryotes have only one origin
Mamallian cells divide about 15 - 30 hours

Base pairs can recover from bubbles and need to keep it open.

SSBP
[[Single Stranded Binding Proteins]], or SSBP, binds to backbones of separated starands to maintain gap.

SSBP might cause nucleo bases to pair with others on the same strand. It inhibits inter and intra strand binding.

SSBP Looks for single stranded dna 

[[Deoxyribonucleic Acid Polymerase]], there is a lot in the eukaroytic cell, 5 in prokaryotic.
It is similar to all organisms with a little variation.

[[Deoxyribonucleic Acid Polymerases I|DNA Pol I]]
[[Deoxyribonucleic Acid Polymerases II|DNA Pol II]]
[[Deoxyribonucleic Acid Polymerase III|DNA Pol III]]

I and II are [[Repair Polymerases]]
[[Replicative Polymerases]]
I is during replication
II is after replication
III is Replicative, involved with replicating genomic dna replication

All have DDDP 
DNA Dependent DNA polymerase
needs a template to make more DNA

Polymerases make DNA.

Denovo means it needs a primer to replicate
Replication goes through 5' - 3' direction

[[Primers]] is a small [[Oligopeptides]] , 8 - 10 bases in length

Phosphate on 5' and OH on 3' 
OH is needed

[[Deoxyribonucleic Acid Dependent Deoxyribonucleic Acid Polymerases]] 
[[Ribonucleic Acid Dependent Ribonucleic Acid Polymerases]] 
Active Sites of DDDPs

1. dNTP (all 4)
2. Templates
3. Primer

DNA Polymerase is a enzyme

Where are [[Primers]] coming from?
Comes from enzyme called [[Primase]] and puts on a RNA primer
Primer needs to join anti parallel to the fork
DNA P uses DNA Pol III to replicate

Daughter strand is the novel strand?

[[Leading Strands]] that leads [[Polymerization]] 
Primers need to be removed because RNA should be mixed with DNA
Then repaired with DNA Pol I

[[5'-3' Exonuclease Activity]] works at the end of the primer

[[Okazaki Fragments]] are small gaps that appear from the removal of RNA, which are fixed with [[DNA Pol I]]

[[Phosphodiester Linkages]] cannot be completed with out [[Deoxyribonucleic Acid Ligases]], and it ligates the phosphates together

Repeated connection of primer can cause problems

[[Processivity]] DNA poly should be processive, the binding of a DNA polymerase to the primer template followed by incorporation of Watson Crick Base Paired dNTP, successively for very long stretches before primer template dissociation.

[[Distributive]] DNA are temporary

[[Processivity Factor]] have the primer and tmplate in the doughnut shape.

Prokaryotes have [[Beta Clamps]]
Eukaryotes have [[Proliferative Cell Nuclear Antigen]]

Have a higher change of mutation called [[Frameshift Mutations]], avoided by keeping primer on template

# 11/01/2023 Lecture

## DNA Replication

[[Origins of Replication]] or ORI
[[Helicase]] denatures DNA Strands breaking hydrogen bonds
[[Replication Bubbles]] making [[Replication Forks]]
[[Single Stranded Binding Proteins]] recognize phosphate backbones and keeps it more linear
[[Deoxyribonucleic Acid Polymerase]] has DDDP activity 
Doesn't add ntps, but dNTPS
Cannot be [[De Novo]]
Directional, initaite points start at -OH 3' 
[[Ribonucleic Acids|RNA]] Primers 
[[Primase]] makes RNA Primer 8 - 10 base pairs
[[DNA Pol I]] Repairs genomic and [[5'-3' Exonuclease Activity]]
[[DNA Pol II]] Repairs post replication 
[[DNA Pol III]] Replicates
All polymerases have 5' - 3' activity
All of them can incorporate [[Deoxynucleotide Triphosphate|dNTP]]
[[DNA Pol III]] attaches around 3' of primer 
Moving towards the [[Replication Forks]] is the [[Leading Strands]]
[[Helicase]] moves along and open up the strands until it hits a new bubble
[[Parent Strands]] vs [[Daughter Strands]]
Synthesis stops when meeting a primer creating a [[Nicked Strand]]
[[DNA Pol I]] detects [[Nicked Strand]] and removes [[Primers]]
Removal of [[Primers]] creates single strand gaps
[[DNA Pol I]] fills up the gaps
[[Okazaki Fragments]] are being created from removing primers
[[Lagging Strands]] follows [[Leading Strands]]
Truncated proteins causes mutation, aggregate, mess up function and shape, and interupted with [[Polyubiquitination]]
[[Proliferative Cell Nuclear Antigen]] works like a zipper to let dNTP to pairs
Processive vs Distributive Enzymes

[[Primer Templates]] 
[[Fidelity]] should be high to be flexible, can cause [[Evolution]]
Proofreading and Editing Mechanism
DNA POL I has [[3'-5' Exonuclease Activity]] to edit dNTP sequences

## The Replisome

[[Replisomes]] 
[[Nick Translations]] 5 - 3' exonuclaease activity
[[Proliferative Cell Nuclear Antigen|PCNA]]

## Proof Reading and Editing by DNAP

[[DNA Pol II]] 3 - 5' activity
Replication is Bi-Directional
[[Topoisomerases]], [[Type I Topoisomerases]], [[Type II Topisomerases]], [[Deoxyribonucleic Acid Gyrase]]

Topo I in Rep, has reactive [[Tyrosine]], it swivels around 
Nick DNA at Backbond
Bind DNA so it swivels
Religate it to seal up backbone

Topo II Recomb, both strands
DNA Gyrase is like TOPO II in Prokayrots 

- [x] CHEM 111 Homework âœ… 2023-11-09
- [x] BIOL 140 Homework âœ… 2023-11-14
- [x] PH 130 QUIZ 1 âœ… 2023-11-05
- [x] PH 130 QUIZ 2 âœ… 2023-11-05
- [x] PH 130 QUIZ 3 âœ… 2023-11-05

[[Satellite DNA]] is repetitive 
[[CPG Islands]] [[Z-Deoxyribonucleic Acids]] interaction
Micro vs Macro Satellites at the end of chromosomes
[[Telomeres]] discovered by Barbara McClintock and also [[Transposable Elements]]
Ends of dna of certain corn had problems
Missing Genes
Elizabeth Blackburn
Carol Grider
[[End Replication Problems]]
[[Telomeres]] will interact to fix end replicaition
[[Teleomerase]] is made of [[Human Telomerase Reverse Transcriptase]] and [[Human Telomerase Ribonucleic Acid]]
Human Terlomerase Reverse Transcripase
found in Retrovirus 
HIV -> Single strand RNA Virus and needs to change in to dsDNA
RDDP Activity
RNA dependent DNA Polymerase
DDDP Activity
DNA dependented DNA Polymerase

# 11/06/2023

## Telomerase

[[Telomeres]] 
A large [[Proteins]] RNA Complex
The [[Ribonucleic Acids|RNA]] contains a templating sequence for synthesizing new DNA Telomere repeasts
The synthesis reaction is done by the [[Reverse Transcriptase Domain]] of the protein

[[Teleomerase]] carries its own RNA template with it at all times

Telomerases are [[Enzymes]]
Are like [[Reverse Transcriptase]] like in viruses like HIV
It doesn't need a substrate to work
[[Central Dogma of Molecular Biology]] is DNA RNA PROT
Reverse transcription is ssRNA, dsDNA, ssRNA, Prot
Humans have two components of Telomerase
	[[Human Telomerase Reverse Transcriptase]] is a protein component
	[[Telomere Reverse Transcriptase|TERT]]
	[[Human Telomerase Ribonucleic Acid]] Human Telomerase RNA
[[Telomeres]] are at the end of the chromosomes
Getting chewed up is [[Crisis]], but before that happens it is [[Senescense]]
Cell can still metabolize but not die soon


[[Coding Sequences]]
[[Satellite DNA]] are repeats 
Telemores are [[Macro Satellites]]
[[CPG Islands]], Cytosine Phosphate Guanine

[[Human Telomerase Reverse Transcriptase|]]
[[Human Telomerase Ribonucleic Acid|Human Telomerase RNA]] is RNA
End of chromosome acts as primer
HTR has an area of 6 bases which are templates for telomeres

Translocate by 6 bases
HTR works as a template
We don't know how telemareses stop at every 6

Cancers are 95% telmares positive
It is a marker of cancer
Telomerase production goes up
[[STEM Cells]]
[[Germ Cells]]
[[Cancer Cells]]
[[Somatic Cells]] are all cells except [[Germ Cells]], and will not express [[Teleomerase]]
[[End Replication Problems]] causes uneven strands
5' prime exonuclease 

Telomares Function 

## Theta Replication

[[Theta Replication]] is a directional process, it looks like Theta symbol during process

Prokaryotes have circular DNA

[[Rolling Circle Replication]] double stranded circular DNA
Out strand gets nicked and removed
Inner strands are templates

Viruses can keep going around in circles in one strand
Cleavage sites
[[Concatamers]] 
What is the mode of replication 

## Modes of a DNA Replication

[[Conservative Replications]] takes a lot of energy for parents to create daughter strands, it is convservative because everything is conserved 

[[Semi-Conservative Replications]]
Parent strands are equally distributative

[[Dispersive Replications]]
Checkered interlace between all strands

DNA is labelled with N-14 to label bases
N-15 in from of heavy ammonia, molecular weight goes up \
Gradient would use CsCl and then [[Cell Fractionation]]

# Meselson-Stahl esxperiment

[[Meselson-Stahl experiment]]

Packaging the DNA with group of proteins called [[Histones]] and DNA make [[Nucleosomes]]
[[H2a]]
[[H2b]]
[[H3]]
[[H4]]
The previous are called [[Dimer]]
DNA wraps around an

[[H1]] is a linker Histone, it links the other histones during DNA production

Nucleomes are involved in RNA transcription
There are a few different type of enzymes that can be involved we theses

[[HATs]]
	Histone Actyl-Transferase
	Actyline will be added to lysine
[[HDACs]]
	Histone deactyl transferase

Sperm productions histones get replaced with [[Protamines]]
Has ordered array of [[Toroids]] which is believed to exit the same way in the egg
It creates a pro nuclei and mother nuclei
Pro nuclei is larger
[[Transgenic]] Animals to add genomic material to cell itself
[[Protyoncogene]] can cause unregularited micosis and tumorogenesis

## Central Dogma

What is [[Ribonucleic Acids|RNA]]
It is a n intermediate, takes info from DNA to the [[Ribosomes]] to make proteins
[[Messenger Ribonucleic Acid|mRNA]] messenger RNA has info from genes
[[Transfer Ribonucleic Acids|tRNA]] Transfer RNA transfer amino acid to cytoplasm
[[Ribosomal Ribonucleic Acids|rRNA]] ribosomal rna and protein makes ribosomes 

[[Ribonucleic Acids|RNA]] is single stranded but can form pairs 
Creates secondary and tertiary strucuters
and create [[Stem Loop Sequences]]
Interupted repeats with any nucleotite in the loop

[[Ribonucleic Acids|RNA]] uses [[Uracil]] instead of [[Thymine]]
Sugar is [[Ribose]] with a [[Hydroxyl Groups]] at 2' prime position
made up of [[Nucleotides]] 
single stranded
uracil
some rna has inherent catalytic activity

[[Ribozymes]] is a catalytic molecule
Has shorter chains of [[Nucleotides]]
RNA initiates at a specific part called [[Promoters]] is a [[Regulatory Regions]] 
[[5' Untranslated Region|5' UTR]] 5' Untranslated Region
[[Exon I]] coding uses correct amino acids
[[Intron I]] non coding and interupts coding sequence from genes
There can have multiple Exon and Introns
[[3' Untranslated Region|3' UTR]]

And the end promotor the first base is a +1, +2, +3, +4 ongoing on gene
[[Downstream Regions]] Positive intergers
[[Upstream Regions]] Negative Integers
There is no 0

mRNA structure

RNA has to be processed 
[[RNA Processing]]
- Placement of 5' Cap
- Very end of 3' [[Poly-A Tails]], prokaryotes dont have it
- [[Ribonucleic Acid Splicing|RNA Splicing]] removal of intron sequence
	- Prokaryotes do not undergo splicing
- mRNA -> 5' Cap -> 5' UTR -> Exons -> 3' UTR -> Poly A Tail

Promoter is not transcribes, but iniiates
Exon linkage are called [[Open Region Frame]] or [[Open Region Frame|ORF]] viruses don't have room for seuqneces in genome, they don't have intonrs
prokaryotes don't have introns

[[Modified Nucleobases]] have an added funcional group 
Can be [[Catalysts|Catalytic]] 

[[Ribosomal Ribonucleic Acids|rRNA]] is the most abundant part of the cell
Catalytiic Componenet
contains different rnA Molecules
18S
5.8 S
28 S
and 5S rRNA

[[Svedber Units]]
The integers are referring to depth of molecules found when centrifuged, 18mm in 18 S

Majority of [[Ribosomal Ribonucleic Acids|rRNA]] are formed in the [[Nucleolus]] high transcriptional activity
Ribosomal proteins combine with rRNA in nucleolus, eventually formed in [[Cytoplasm]]

[[Unprocessed mRNA|hnRNA]] or Heterogenous RNA

Ribosomes bring all major RNA 

[[Polycistromic]]

Prokaryote regulate ABC 
Poly means many

Cistron is similar to genes
initiate transcription using [[Ribonucleic Acid Polymerases]]

DDRP
[[Polysomal RNA]]
[[Coupled Synthesis]]

# 11/13/2023 

Telomase mutation
Is it affected? No
[[Human Telomerase Ribonucleic Acid|hTR]] is a template
Is not affected by mutated [[Nucleotide Triphosphate|NTPs]]
They code for nothing

More bacteria produced
With increase of radiation

$\ce{ ^35S }$ is protein
$\ce{ ^32P }$ 

## Poly-A Polymerase

[[Poly-A Tails]]
Poly A Sequence
Bind Sites
Cleaved at `AAUAAA`

How does [[Polyadenylation]] regulate?
Protects messages from 3' RNAses

No poly a tail, RNAses eat it

## Splicing via the Splicesome


[[Small Nucleolar Ribonucleic Acids Proteins|SnRPs]]
[[Splicesomes]]

[[Exon I]] and [[Intron I]]
Donor 5' end and Acceptors 3' end

[[uSnRPs]]
[[U1]] and [[U2]]
comes together and interact with complex of nother SnRPs

$\ce{ E_{1}, I_{1}, E_{2}, I_{2}, E_{3}, I_{3} }$

[[Constitutive Splicing]]  
[[Alternative Ribonucleic Acid Splicing]] 

Can be spliced into different [[Isoforms]]

# Secondary Structures of RNA

[[Secondary Structures]] of RNA

[[Hairpins]]
[[Stacks]]
[[Bulges]]
[[Multiloop]]
[[Internal Loop]]

[[Transfer Ribonucleic Acids|tRNA]]

- [x] Read and Review BIOL 140A Chapter 16 âœ… 2023-11-27
- [x] Read and Review BIOL 140A Chapter 17 âœ… 2023-11-27
- [x] BIOL 140A Midterm ðŸ“… 11/27/2023 âœ… 2023-11-27
- [x] BIOL 140A Homework CH 16 â« âœ… 2023-11-27

# 11/15/2023 Lecture

## Proteins

[[Filamentous Proteins]]
[[Globular Proteins]] - anything that isn't [[Filamentous Proteins]]
[[Electrostatic Models]]
White is neutal
Blue Positive
Red Negative

[[Ribbon Models]] amino acids 
Knowing the amino acids will understand 
Alteret the protein to understand function change
Filamentous has high tensile strength

[[Dehydration Synthesis]] take out an H2O to form [[Peptide Bonds]]
[[Polypeptide Backbones]] forms the structure

[[Mutations]] within active sites
M184V
Met on 184
Wild type > Valine

Changing [[Potential of Hydrogen|pH]] changes molecule charge

$\ce{ gly-ala-glu-lys-arg-g\ln-asn-ser }$

[[Oligopeptides]] is up to 20 [[Peptides]]

[[Primary Structures]] can be [[Oligopeptides]] and [[Polypeptides]]
[[Chaperone Proteins]] guide the chains
[[Disulfide Bridges]] are strong and can be only broken down with Dtt, beta meraptoa ethonol
[[Reducing Agents]] can break disulfide bonds, but not other covalent bonds

[[Secondary Structures]] are [[Alpha Helices]] which are stabilized by [[Hydrogen Bonds]] between the [[Polypeptide Backbones]]

[[Serpentine Proteins]] goes in and out of [[Proteins]]
[[Ligands]] bind to [[Serpentine Proteins]] 
Signals can end at a [[Cell Nucleus]] or outside of it

[[Beta Pleated Sheets]] are less rigid when are parallel
[[Beta Barrels]] excess or lack of water can destroy cell
[[Aquapores]]

[[Receptors]] have long tails
[[Phosphorylation]] usually means activate deactive

[[Kinases]] = [[Phosphorylation]]
[[Phospateses]] = de [[Phosphorylation]]

[[Kinases]] [[Phosphorylation|Phosphorylates]] and releases something liek a [[Transcription Factors]] 

[[Immune Specific Expressions]]
[[Serpentine Proteins]]
[[Serpentine Receptors]] mutations can change its function

[[Plasmids]] can inject sequences and alter

[[Conservative Mutations]] conserves same characterists
non [[Conservative Mutations]] 
 
[[Tertiary Structures]] 

Fernomoners

[[Quaternary Structures]] bring in [[Multimer]] and makes [[Multimer|Multimeric]] [[Proteins]] 

## The Genetic Code

How many nucleo tides are necessary to encode an amino acid
There are 20 amino acids
The four bases

[[Messenger Ribonucleic Acid|mRNA]] 

$\ce{ AUG UUU ACA }$

$\ce{ 4^2 = 16 possible codons }$
$\ce{ 4^3 = 64 possible codons }$

each three bases is a [[Codons]] and it codes for an [[Amino Acids]]

[[Degenerate]] 
[[Stop Codons]]

$AUG$ Methionine Initiation Codon is the first amino acid on the protein
$\ce{ Met_{i} }$ for eukaryotes
AUG AUG
Met fernylated (pro)

1 codon is one amino acid

[[Silent Mutations]] changes [[Genotypes]] but doesn't affect [[Phenotypes]]

[[Stop Codons]] are like period in a sentences
UUA - named after daughter Amber
UAG - Ochre

UGA - 

[[Marshall Nirenberg]] [[Johann Matthaei]]

Used a cell free system

Poly-u makes all [[Phenylalanine]]

UUUUUUUUU - makes phenyalanine

[[Transfer Ribonucleic Acids|tRNA]] transfer RNA is the most modified molecule in the cell

[[Clover Leaf]] 

Cy T Loop

[[Psuedouranine]]

[[D Loop]]

[[Transfer Ribonucleic Acids|tRNA]] structures


[[Aminoacyl-Transfer Ribonucleic Acid Binding Site]] put on with [[Aminoacyl-Transfer Ribonucleic Acid Synthetases]] 5' 
[[Anticodon Loops]] is complementary

[[Messenger Ribonucleic Acid|mRNA]] 

[[Open Reading Frames]] have nothing but [[Codons]]

[[Adenosine Triphosphate|ATP]] convers into Aminoacyl-tRNA

Translational factor occurs

[[Ribosome Composition]]

Large and Small are same in pro and euk
different structure

70S

80S

Sedimentation Coefficient

Eukaryotic cytoplasmic ribosomes are larger and more complex than prokaryotic ribosomes

mRNA binds small and large subunits and interacts with 5'caps

AUG codon lines up with EPA pockets


[[Exit Pockets]] 
[[Peptidyl-Transfer Ribonucleic Acid Binding Site]] contains [[Peptidyal Transferase]]
[[Aminoacyl-Transfer Ribonucleic Acid Binding Site]]

## Ribosomes

[[Ribosomal Subunits|Large Subunits]]

## Structure of the e. coli ribosome

Small unit reads mrna

[[Initiation]]

[[Elongation]]

[[Transcription]] and [[Translation]]

[[Shine-Delgarno Sequences]] in prokaryotes binds with 16S rRNA
[[Kozak Sequences]] in eukaryotes binds with 18S rRNA 

[[Codon-Anticodon Interactions]]

Condon anticodon base pairing is antiparallel
third position in codon is frequently degenerated
one tRNA can interact iwth more than one codon

[[Wobble Rules]] 

[[Hypoxanthine]] is a Pre-Cursor Base and is added to a [[Ribose]] and makes [[Inosine]], alternate hydrogen bonding occurs and makes wobble positions

[[Internal Ribosome Entry Sites]] IRES

Polysystronic 

# Cal State East Bay 2 Recording from Martha

[[Glycolysis]] gets two [[Pyruvate]] 
[[Pyruvate]] injects into [[Mitochondria]] using 2 [[Adenosine Triphosphate|ATP]]
[[Pre-Citric Acid Cycle]] creates converts [[Pyruvate]] to [[Acetyl Coenzyme A|Acetyl CoA]]
[[Nicotinamide Adenine Dinucleotide|NAD+]] becomes [[Nicotinamide Adenine Dinucleotide|NADH]]
It makes a waste product, CO_2
Takes two oxygens from pyruvate
CO2 exchanges with O2 from Red Blood cells
[[Oxaloacetate]] is the first step of the krebs cycle

## Formation of Citrate

[[Acetyl Coenzyme A|Acetyl CoA]] gets condesed with [[Oxaloacetate]] to form [[Citrate]] and its catalyzed by [[Citrate Synthase]]
[[Coenzyme A]] gets recycled

## Formation of Isocitrate

[[Citrate]] is rearranged -> [[Isocitrate]] by [[Acontinase]]

## Oxidation of Isocitrate to a-Ketoglutarate

[[Isocitrate Dehydrogenase]] catalyzes oxidative decarboxylation of [[Isocitrate]] to form [[Alpha-Ketoglutarate]]

## Oxidation of [[Alpha-Ketoglutarate]] to [[Succinyl-Coenzyme A]]

[[Alpha-Ketoglutarate Dehydrogenase]] catalyzes NAD+ oxidation to NADH

## Conversion of Succinyl-CoA to Succinate

[[Coenzyme A|CoA]] is removed from [[Succinyl-Coenzyme A]] to produce [[Succinate]]
The Energy released is used to make guanosine triphosphate from Guanosine Diphosphate and P through Substrate Level Posophorylation
[[Succinyl-Coenzyme A Synthase]] is the catalyst

[[Flavin Adenine Dinucleotide|FAD]] is like [[Nicotinamide Adenine Dinucleotide|NAD+]]

## Oxidation of Succinate to Fumarate

[[Succinate]] is oxidized to [[Fumarate]] and [[Flavin Adenine Dinucleotide|FAD]] is reduced to [[Flavin Adenine Dinucleotide|FADH2]]
This is catalyzed by [[Succinate Dehydrogenase]] removes two hydrogen from succinate

## Hydration of Fumarate to Malate

Reversible hydration of fumarate to L-Malate is catalyzed by [[Fumarate Hydratase]] H_2O is added to fumarate to make [[Malate]]

## Oxidation of Malate to Oxaloacetate

[[Malate]] + HAD+ -> [[Oxaloacetate]] + NADH

## Overall TCA Reaction

$\ce {Acetyl CoA + 3 NAD^+ + FAD + ADP + HPO_4^2- -> 2 CO_2 + CoA + 3 NADH^+ + FADH_2 + ATP}$

12 total ATP
3 NAD+ = 9 ATP
2 FAD = 4 ATP
1 ATP = 1 ATP

Used in anabolic processes

## Electron Transport Chain

10,000's of this throughout the membrane

