---
datecreated: 2023/11/28
aliases: 
course:
---
# CRISPR

Paolo Canuto, Yayra A, Maya C, Britney L
Jennifer Lloyd
BIOL 140A
8 December 2023 

# Introduction

Objective of the experiment is use [[Clustered Regularly Interspaced Short Palindromic Repeats-Cas9 System|CRISPR-Cas9 System]] to edit the bacterial gene of E. Coli that enables it hydrolyze lactose and lactose analogue. The technique of [[Bacterial Transformation]] will be used along with two different plasmids, pD and pDG. [[pD Plasmid]] contains a DNA sequence that repairs DNA strands with [[Homology-Directed Repair]] and a codon to create a nonsense mutation on the lacZ gene. [[pDG Plasmid]] is similar to pD, but contains a sequence for [[sgRNA]] which guides Cas9 to cleavage sites.

# Materials and Methods

## Part 1A Bacterial Transformation with pD and pDG Plasmids

Acquired four 1.5 mL microcentrifuge tubes and lablled as A, B, C, and D. Added 250 uL of cold transformation solution to each tube and placed immediately in ice.

Using a sterile loop, acquired 5 IPTG/X-gal bacterial colonies in both Plate A and B; mixed thoroughly and made sure to use new loops per tube.

Using a new loop, transferred 5 colonies of IPTG/X-gal/Ara to each B and C; mixed thoroughly using new loops per tube.

Transferred 10 uL of pLZDonor plasmid to Tubes A and C, flicked three times and iced. Transferred 10 uL of pLZDonorGuide to tubes B and D, flicked three times and iced. Incubated tubes on ice for 10 minutes.

Transferred tubes to heat block at 60 C for 50 seconds, afterwards placed tubes in ice for 2 minutes. Added 250 uL of LB Nutrient Broth to each tube and incubated at room temperature for 20 minutes.

## Part 1B: Plating of CRISPR Samples

Acquired four mini IPTG/X-gal/spectinomycin agar places and labeled as the following: IX/SPT/Sample A/12-28-2023, IX/SPT/Sample B/12-28-2023, IX/SPT/Sample C/12-28-2023, IX/SPT/Sample D/12-28-2023.

Agitated each sample and transferred 100 uL to their respective plates, making sure to use new tips, used new sterile loops for each and used the streak plate method. Parafilmed each plate to and stacked gel side up, taped all plates together and placed incubator.  

## Part 1C: Counting Bacterial Colonies and Data Analysis

Transferred samples from incubators to bench, and most likely was counted. Data provided by Britney and Maya via Canvas.

# Results

| Plate | Blue Colony Qt | White Colony Qt | White Colony % | lac Z Change      |
| ----- | -------------- | --------------- | -------------- | ----------------- |
| A     | 8              | 112             | 93.3           | Mostly Functional |
| B     | 0              | 140             | 100.0          | nonFunctional     |
| C     | 40             | 12              | 23.0           | Mostly functional |
| D     | 0              | 52              | 100.0          | NonFunctional     |

**Table 1:** Number of White vs Blue colonies present in incubated agar places, raw data provided by Britney L and Maya C.

Much more prominent difference observed in plate C when compared to the rest of the plates.

# Discussion

Plates B and D shows evidence of nonfunctional lacZ proteins through the observation of all white colonies. Plate A and C shows evidence of functional lacZ enzymes through the presence blue colonies, which are more prominent in plate C. 

Based off the data shown above, the plates with pDG were effective at breaking the lacZ gene with the help of the guide RNA that pDG codes for vs pD. 

The data is inconsistent with the theoretical results from the lab manual and Out of the Blue CRISPR Kit Instructor Guide. If there was a lack of ARA in plate B, pDG should be still able to cut the gene but not repair, thus no growth should be possible.

Plates A and C should technically be all blue colonies since if there was a lack of a sgRNA to guide Cas9 to cleave a few genes. Considering the numbers of blue colonies relative to white, there is evidence of affected lacZ where there shouldn't.

Results not surprising considering some incubation in ice was much longer from the lack of attention to time. 

Redoing the experiment is recommended, there might have been cross contamination or ineffective techniques performed during experiment.

# Conclusion

With the results provided by peers, the data shows that the controls exhibit gene editing where there should not be. With the lack of Arabinose sugar, the HDR in plate B should be produce thus leading to an absence in colonies. Either the wrong results were sent, or procedure was done incorrectly. This experiment is inconclusive.

# Reference

Lloyd, J. (2023, April). Laboratory Manual for Principles of Cells and Molecular Biology (J. Lloyd, Ed.) [Review of Laboratory Manual for Principles of Cells and Molecular Biology]. Jennifer Lloyd. https://dl.orangedox.com/labmanual

Bio-Rad. (n.d.). _Out of the Blue CRISPR Kit Instructor Guide_. https://www.bio-rad.com/webroot/web/pdf/lse/literature/10000122375.pdf

‌