# Green Biome Institute GBI Project

Paolo Canuto/Maya Caple
Jennifer Lloyd
BIOL 140A
11/09/2023

# Introduction

Preparation of Lilium Candidum for a [[DNA Sequencing Library]] will be conducted through the standard procedure of DNA Isolation, Adapter Ligation, Nick-Repair, and Purification of Ligated DNA, and Gel Electrophoresis. 

# Materials and Methods

**Part 1: DNA Isolation and Purification Protocol for Plant Matter** (QIAGEN, 2018)

Acquired a 1.5mL microcentrifuge. Using a mortar and pestle, muddle the lily plant matter with **400 μL of buffer AP1** in addition to **4 μL of RNase A** (which has a stock concentration of 100 mg/mL). Vortex the solution until no plant tissue remains.  

Incubated the tube at 65 °Celsius for 10 minutes, inverted the tube two to three times throughout. Added 130 μL Buffer P3. Mix well and place the tube on ice for 5 minutes.  

Centrifuged the sample for 5 minutes at 6,000 x _g_.

Acquired a purple QIAshredder mini spin column and transferred the lysate solution onto the white disc of the spin column. Centrifuged the spin column at 10,000 x _g_ for 2 minutes.  

Transfer the flow-through to a new 1.5 mL microcentrifuge tube _labeled_ as plant lysate MC along with your initials.  

$$\ce{ 350 \mu l of DNA * 1.5 = 525 \mu l AW 1 Buffer }$$
$$\ce{ 350 \mu l DNA + 525 \mu l AW 1 =  875 \mu l solution}$$

Into the 1.5 mL microcentrifuge tube which contains the plant lysate added 525 ul AW1 buffer

Next, acquire a white DNeasy spin column and _label_ the lid. Transfer 650 μL lysate solution onto the white disc of the spin column. Centrifuge the spin column (as it sits within the collection tube) at 10,000 x _g_ for 1 minute and dumped flow through

Onto the white disc of the spin column, added 500 μL AW2 buffer. Centrifuge at 6,000 x _g_ for 1 minute. Dumped the flow-through liquid.   

Onto the white disc, added another 500 μL AW2 buffer. Centrifuge again at 6,000 x _g_ for 2 minutes. Dumped the flow-through liquid again. 

Centrifuged at 6,000 x _g_ for 2 minutes. On an accuSpin Micro 17 (by Fisher Scientific), the input will be 6.0 x _g_ (x 1,000) for 2 minutes. Dump the flow-through liquid. 

Acquired a fresh 1.5 mL microcentrifuge tube and _labelled_ the lid. Removed the spin column without touching the bottom portion of it and positioned the spin column into the 1.5 mL tube.  

Added **50 μL buffer AE** to spin column and incubated at room temperature (RT) for 5 minutes. Centrifuged at 6,000 x _g_ for 1 minute. 

Checked concentration of DNA sample with NanoDrop.

Froze the _labeled_ tube at -20 °C.

**Part 3B: Preparation of PCR Samples and Polymerase Chain Reaction (PCR)**

Created PCR sample with 10 μL “2X PCR master mix” (from ProMega), 2 μL of primers (includes 1.0 μL forward primer + 1.0 μL reverse primer), 5 μL of your purified DNA sample, 3 μL ultra-pure water.

Processed in PCR Thermocycler with following conditions:

| PCR Thermocycler Conditions (Green Biome Institute Project) |                 |            |            |
| ------------------------------------------------------------------ | --------------- | ---------- | ---------- |
| **Function**                                                       | **Temperature** | **Time**   | **Cycles** |
| Initial Denaturation                                               | 95 °C           | 2 minutes  | 1          |
| Denaturation                                                       | 95 °C           | 30 seconds | 40         |
| Annealing                                                          | 50 °C           | 30 seconds |            |
| Extension                                                          | 72 °C           | 3 minutes  |            |
| Final Extension                                                    | 72 °C           | 7 minutes  | 1          |
| Hold                                                               | 4 °C            | ∞          | 1          |

 Upon completion of the thermocycler run, the PCR samples will be frozen (at -20 °C) until the next laboratory session.

# Agarose Gel Electrophoresis of Plain Pure Plant DNA

Performed Agarose Gel Electrophoresis of pure plant DNA and standard MWW and 1% Agarose Solution.

Ran Gel at 100v for 30 minutes, documented gel.

## Part 3C: Adapter Ligation, Nick-Repair, and Purification of Ligated DNA (Thermo Fisher, 2016)

Acquired 0.2 mL PCR Tube and using a p200 added 20 uL genomic DNA, "Unknown Sample M.C.", and 5 uL of Adapter Ligation Reagents (ligase buffer, adapters, dNTP mix) nuclease-free water, DNA ligase, and nick repair polymerase). Labelled tube with "PC"

Placed labelled PCR tube into a Thermo Cycler machine with the the following Parameters. 
  
### Thermo cycler Conditions for Incubation (Green Biome Institute Project)

| **Function** | **Temperature** | **Time**   | **Cycles** |
| ------------ | --------------- | ---------- | ---------- |
| Hold         | 25 °C           | 15 minutes | 1          |
| Hold         | 72 °C           | 5 minutes  | 1          |
| Hold         | 4 °C            | ∞          | 1          |

After completion of thermo cycler, transferred Adapter Ligation Sample to new 1.5 mL Microcentrifuge Tube labelled "PC".  

Added 30 uL of AgencourtTMAMPureTM XP bead suspension to 25 uL of ligation sample.
Transferred new mixture to centrifuge at 6.0 x g for 5 minutes. Placed tube in to Sergi Lab Supplies Magnetic Rack for 3 minutes. Using a p200, discarded supernatant while avoiding bead pellet.

Calculated a 70% Ethanol Solution using the following dilution equation:

$$\ce{ 1.0V_{1} = 0.70(500 uL) }$$

Added 350 uL of 100% Ethanol to 150 uL distilled water to create 70% Ethanol Solution.

With the tube still attached to the magnet, transferred 500 μL of freshly prepared 70% ethanol into your sample. Incubated the tube for 30 seconds and inverted the tube three to four times throughout. Once the solution presented clear, removed supernatant with p1000 while avoiding cell bead pellet.

Re-added 350 uL of 100% Ethanol to 150 uL distilled water to create 70% Ethanol Solution.

With the tube still attached to the magnet, transferred 500 μL of freshly prepared 70% ethanol into your sample. Incubated the tube for 30 seconds and inverted the tube three to four times throughout. Once the solution presented clear, removed supernatant with p1000 while avoiding cell bead pellet.

Centrifuged sample at 10.0 x g for 1 minute to remove excess ethanol. Placed it back into the magnetic rack and Using a p20 micropipette, discarded the supernatant without provoking the bead pellet.

With the tube still attached to the magnet, air‐dry the beads at room temperature (RT) for 5 minutes.

Removed the tube from the magnetic rack and transferred 20 μL of Low TE directly to the pellet to scatter the beads. Mixed with pipette and vortex for 10 seconds.

Centrifuged sample at 10.0 x g for 1 minute the sample and then returned the tube to the magnetic rack for at least 1 minute. Once the solution presented as clear, the supernatant was transferred to a new tube without disturbing the pellet. Labelled as "PC PURE PLANT MC SAMPLE 10/31/23"

Submitted to Jennifer L to freeze the _labeled_ tube at -20 °C.

## Part 4A: Agarose Gel Electrophoresis of Amplified Plant DNA Samples (after PCR) and Adapter Ligation

Measured out 0.5 grams of agarose powder and added 125 mL Erlenmeyer flask along with 50 mL of stock 1X TAE Buffer to make a 1% solution. Microwaved for 60 seconds and swirled every 30 seconds, added 3 uL of GelRed and mixed.

Assembled casting tray and poured GelRed Agarose Gel Solution to about 0.5 cm thickness. After gel solidified, removed barriers, poured 1X TAE to each reservoir until Max level reached, then removed gel comb.  

Using a p20 micropipette, added 5 μL of loading dye and 5 μL of plant PCR sample to a piece of Parafilm and mixed with pipette.

Using a p20 micropipette, added 5 μL of loading dye and 5 μL of Adapter Ligation Sample to a piece of Parafilm and mixed with pipette.

Onto a fresh piece of Parafilm, combined 5 μL of molecular weight marker (MWM) + 2 μL of loading dye (LD) + 3 μL of 1X TAE.  

Using a new p20 micropipette for each sample, transferred 10 uL of each sample from left to right: MWM, ==PCR DNA==, Adapter Ligation DNA.

%% PCR DNA was dispersed on para film, Adapter ligation was using a random sample, would check with Maya to see. %%

Ran the gel for ~45 minutes at 100 Volts.

Took a photograph of the agarose gel (via ChemiDoc and Image Lab software), and documented with Pixel 7a on automatic settings. 

# Results

![[0130CapleCanuto.jpeg|500]]

**Figure 1:** DNA concentration for isolated plant sample

![[Pure Plant DNA.jpeg|500]]

**Figure 2:** Gel for pure dna. (MWM Left Well, Pure DNA Right Well)

![[Green Biome Institute.jpg|500]]

**Figure 3:** Gel for MWM, PCR DNA, and Adapter ligation Sample

# Discussion

Most of the results can use a lot more improvement. DNA concentration of the isolated DNA can be improved in the A260/A280 ratio, but an overall 22.2 ng/ul was satisfactory amount. DNA isolation can be repeated for improved results. An issue that occurred was that the isolated sample was thrown away after preparing a general PCR sample. Experiment's control has been negatively affected, compensated by utilizing other samples from class. 

Kiddy's sample was used to check the contents of bands in the first agarose gel electrophoresis; it exhibited a wide range of molecular bands which is a evidence of a good sample.

Due to lack of original sample, a random sample from a different class was used for the Adapter Ligation sample. Was unable to check concentration due to lack of AE Buffer during that lab session.

Final gel electrophoresis was inconclusive. Original prepared PCR sample didn't have any bands, and the random sample used for Adapter Ligations presented no bands as well. 

# Conclusion

Overall quality of experiment was lost over the course of the 4 sessions. Due to the misplacement of original samples, then utilization of random samples, the experiment didn't provide satisfactory results. Re-experimentation with more control is highly advised. 

# Reference

Lloyd, J. (2023, April). Laboratory Manual for Principles of Cells and Molecular Biology (J. Lloyd, Ed.) [Review of Laboratory Manual for Principles of Cells and Molecular Biology]. Jennifer Lloyd. https://dl.orangedox.com/labmanual


